fsevents.go

```diff
@@ -158,7 +158,9 @@ func (es *EventStream) Start() {
 	// in C callback
 	cbInfo := registry.Add(es)
 	es.registryID = cbInfo
-	es.uuid = GetDeviceUUID(es.Device)
+	if es.Device != 0 {
```

Comment:
I'm not sure if this will need to be revised in light of the changes in #41.

We need to rebase this and test it with multiple versions of Go.